PeriodogramAnalysis/multitaper_spectrum_statistics.py

```diff
@@ -0,0 +1,87 @@
+import numpy as np
+from scipy.fft import rfft, rfftfreq
+from scipy.signal.windows import dpss
+
+
+def multitaper_psd(
```

Comment:
Make sure you test this function, especially cuz you said it is generated code.

Reply:
Have tested it, the algorithm is fine and I replaced this function with mne function, the plots are the same.

Reply:
I don't see the test code

Reply:
Added relevant tests in this commit: [1e2470a]